theano/gpuarray/tests/check_dnn_doc.py

```diff
@@ -0,0 +1,284 @@
+"""
```

Comment:
I don't think we want to be that exhaustive, but I'm not opposed to it.

We mostly want to make sure that what is supposed to work, works.